### PR TITLE
Adding CI

### PR DESCRIPTION
**Pull request by animesh-007**
I was trying to work on #26 using Travis Ci
Screen Shot 2020-05-23 at 9 18 56 AM
Can you review my work?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.7"
+cache: pip
+install:
+  - pip install -r requirements.txt
+  - pip install -r requirements-dev.txt
+script:
+  - make format-check
+  - pylint --output-format=text main.py
```
Comment (hao-fang) [inline .travis.yml:10]: why you lint `main.py` again? it has been already checked in `make format-check`.

Comment (animesh-007) [reply .travis.yml]: Thanks for reviewing it
Now I have removed it.
Is there any other change?

Comment (hao-fang) [reply .travis.yml]: Thanks! This looks good to me but you will need to ask the repo owner @srush to see if this is what he planned.

**Comment by animesh-007** [comment]
I hope it is okay that I have also pushed the changes made by using `make format`.

**Pull request merged**